baseapp/baseapp.go

```diff
@@ -1057,5 +1057,18 @@ func (app *BaseApp) ProcessProposalVerifyTx(txBz []byte) (sdk.Tx, error) {
 
 // Close is called in start cmd to gracefully cleanup resources.
 func (app *BaseApp) Close() error {
+	// Close app.db (opened by cosmos-sdk/server/start.go call to openDB)
+	app.logger.Info("Closing application.db")
+	_ = app.db.Close()
```

Comment:
Does this return an error? If so, why do we ignore it instead of returning it?

Reply:
it should return.  must have been bad rebase

Reply:
the goal is to close 2 database handles, but we don't want the first close error to return (missing the second close)

do you have a convention for returning multiple errors?

Reply:
cockroaddb/errors did not have Join() so used golang errors aliased to goerrors
`go test` shows zerolog parse errors regardless of what baseapp.Close() returns (nil or err)

```
zerolog: could not write event: cannot decode event: invalid character '\n' in string literal
zerolog: could not write event: cannot decode event: invalid character '\x1d' in string literal
zerolog: could not write event: cannot decode event: invalid character '\x05' in string literal
```

Reply:
Where is that logging coming from? I think the error from `baseapp.Close()` is discarded anyway in server.

Reply:
```
git clone cosmos-sdk
cd baseapp
go test 
```

prints the zerolog output

Reply:
This issue is prior to your PR as I see it happening on main, so this is unrelated.

Reply:
stlid `errors.Join` should suffice

Reply:
stdlib errors shadowed by import "github.com/cockroachdb/errors"